packages/runtime/src/helpers/files.ts

```diff
@@ -356,7 +356,7 @@ export const getDependenciesOfFile = async (file: string) => {
     return []
   }
   const dependencies = await readJson(nft, 'utf8')
-  return dependencies.files.map((dep) => resolve(file, dep))
+  return dependencies.files.map((dep) => resolve(dirname(file), dep))
```

Comment:
This fixes the tests, i.e. snapshots below, but do we have something in place to test this aside from the snapshot?

Reply:
@nickytonline I added tests in #1780, which is a more comprehensive fix